apps/files/lib/Command/ScanAppData.php

```diff
@@ -239,7 +239,7 @@ protected function showSummary($headers, $rows, OutputInterface $output) {
 	protected function formatExecTime() {
 		$secs = round($this->execTime);
```

Comment:
```suggestion
		$secs = (int)round($this->execTime);
```

To make psalm happy :elephant: